lib/elixir/src/elixir_def_overridable.erl

```diff
@@ -1,6 +1,6 @@
 % Holds the logic responsible for defining overridable functions and handling super.
 -module(elixir_def_overridable).
--export([setup/1, overridable/1, overridable/2, name/2, super/2, store_pending/1,
+-export([setup/1, overridable/1, overridable/2, name/2, kind_and_name/2, super/2, store_pending/1,
```

Comment:
Actually we don't, good catch! We don't need `name/2` at all, I just removed it. We could go even further and make `name/3` take a `Count` instead of the whole `Overridable` map because we always have that count available, but I think for now it's fine like this, `name/2` is nice and scoped like this. Wdyt?